src/core/client/admin/routes/Moderate/Queue/RejectedQueueRoute.tsx

```diff
@@ -155,11 +156,11 @@ const enhanced = (withPaginationContainer<
 
 enhanced.routeConfig = {
   Component: enhanced,
-  query: graphql`
+  query: resolveModule(graphql`
```

Comment:
What's going on with this?

Reply:
Made obsolete and removed by commit `chore: resolve es6 modules directly in babel-plugin-relay` :-)

Reply:
Still couldn't get it to work using the `eagerESModules` flag. Test would fail, but running it in webpack works!? Will try again when this lands: https://github.com/facebook/relay/pull/3025

Reply:
`resolveModule` shouldn't be needed at somepoint. It's temporarily doing a job that `babel-plugin-relay` should be doing. `resolveModule` is in 99% of the cases hidden inside the framework so you normally wouldn't run into it!